plugins/database/src/db_plugin.cpp

```diff
@@ -7520,8 +7528,8 @@ irods::error db_mod_user_op(
     if ( strncmp( _option, "rmPamPw", 9 ) == 0 ) {
         auth_config ac{};
         if (const auto err = get_auth_config("authentication::pam_password", ac); !err.ok()) {
-            irods::log(err);
-            log_db::warn("Failed to get password configuration - using defaults.");
+            log_db::warn("Failed to get auth configuration. Using default values. [{}]", err.result());
+            ac = auth_config{};
```

Comment:
I'm not so sure about this now... Might be better to have `get_auth_config` handle this on a case-by-case basis and not return an error object.

Reply:
I don't understand. Please elaborate.

Aside from that, should we be falling back to the default config or returning an error? The default config could be very different from what the admin wants.

Reply:
Looking at the previous implementation, it seems that an error was indeed returned in the start operation if any of the configurations fails to be extracted for whatever reason. Will adjust this.

Reply:
Done